File: exam-slammr-webapp/src/test/java/uk/co/claritysoftware/exam/slammr/webapp/web/model/user/SocialUserSignUpTest.java
package uk.co.claritysoftware.exam.slammr.webapp.web.model.user;

import org.junit.Test;

import nl.jqno.equalsverifier.EqualsVerifier;
import nl.jqno.equalsverifier.Warning;
import uk.co.claritysoftware.exam.slammr.webapp.web.model.user.SocialUserSignUp;

/**
 * Unit test class for {@link SocialUserSignUp}
 */
public class SocialUserSignUpTest {

	@Test
	public void shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(SocialUserSignUp.class)
				.suppress(Warning.NONFINAL_FIELDS)
				.verify();
	}

}

File: exam-slammr-webapp/src/test/kotlin/uk/co/claritysoftware/exam/slammr/webapp/web/model/user/SocialUserSignUpTest.kt
package uk.co.claritysoftware.exam.slammr.webapp.web.model.user

import nl.jqno.equalsverifier.EqualsVerifier
import nl.jqno.equalsverifier.Warning
import org.junit.Test

/**
 * Unit test class for [SocialUserSignUp]
 */
class SocialUserSignUpTest {

	@Test
	fun shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(SocialUserSignUp::class.java)
				.suppress(Warning.NONFINAL_FIELDS)
				.verify()
	}

}